mergify_engine/debug.py

```diff
@@ -189,21 +194,37 @@ def _url_parser(
     )
 
 
+async def get_owner_id_from_login(owner_login: str) -> int:
+    url = f"{config.GITHUB_API_URL}/users/{owner_login}"
+    async with http.AsyncClient() as client:
+        try:
+            installation = (await client.get(url)).json()
+        except http.HTTPNotFound as e:
+            LOG.error(
+                "Can't get owner id from login.",
+                gh_owner=owner_login,
+                error_message=e.message,
+            )
+            raise http.HTTPNotFound
+    return int(installation["id"])
```

Comment:
the owner id is located at `installation["account"]["id"]`

Reply:
actually the issue is the variable name : the variable should be called 'user'